table/block_based/block_based_table_builder.cc

```diff
@@ -505,9 +505,12 @@ struct BlockBasedTableBuilder::Rep {
           use_delta_encoding_for_index_values, p_index_builder_));
     }
 
-    for (auto& collector_factories : *tbo.int_tbl_prop_collector_factories) {
+    const auto& factory_range = tbo.int_tbl_prop_collector_factories;
+    for (auto it = factory_range.first; it != factory_range.second; ++it) {
```

Comment:
Just a question for the future usecase, will the needed properties are all in one range?

Reply:
Yes. For now, we actually just need a way to skip (or not skip) a single (BlobDB specific) factory.